clients/loader.py

```
@@ -0,0 +1,151 @@
+"""Client configuration loader with caching and pre-compiled patterns."""
+import json
+import re
+import logging
+from collections import OrderedDict
+from pathlib import Path
+from dataclasses import dataclass, field
+from typing import List, Dict, Optional, Pattern
```

Ruff (0.14.10)

Import from `re` instead: `Pattern`

Import from `re`

(UP035)